src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -157,7 +158,13 @@ const SettingsAccessLinkManagement = ({
                 description: 'Column header for the link management data table in the settings page.',
               }),
               accessor: 'link',
-              Cell: LinkTableCell,
+              // eslint-disable-next-line react/no-unstable-nested-components
+              Cell: (props) => (
+                <LinkTableCell
+                  {...props}
+                  enterpriseSlug={enterpriseSlug}
+                />
+              ),
```

Comment:
nit: you could likely leave `Cell: LinkTableCell` as is, if `LinkTableCell` were to get the `enterpriseSlug` from the Redux store (similar to `SettingsAccessLinkManagement`).

Reply:
Good idea, but that's just gonna be one more place to rip redux out of eventually, right?